From 3c3e40c72e96b55a98c5fe4514269fe72401e62a Mon Sep 17 00:00:00 2001
From: Sai Vemuri <svemuri@redhat.com>
Date: Thu, 11 Sep 2014 22:42:32 -0400
Subject: [ethernet] cxgb4: Add more PCI device ids

Message-id: <1410475447-94392-14-git-send-email-svemuri@redhat.com>
Patchwork-id: 93794
O-Subject: [RHEL7.1 PATCH BZ 1124947 013/108] cxgb4: Add more PCI device ids.
Bugzilla: 1124947
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Signed-off-by: Hariprasad Shenai <hariprasad@chelsio.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit f0a8e6dea0f6785e84673f5f9ddb620de5447aba)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
index 950dea1..0e7ae06 100644
--- a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
+++ b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
@@ -254,6 +254,8 @@ static DEFINE_PCI_DEVICE_TABLE(cxgb4_pci_tbl) = {
  CH_DEVICE(0x5011, 4),
  CH_DEVICE(0x5012, 4),
  CH_DEVICE(0x5013, 4),
+ CH_DEVICE(0x5014, 4),
+ CH_DEVICE(0x5015, 4),
  CH_DEVICE(0x5401, 4),
  CH_DEVICE(0x5402, 4),
  CH_DEVICE(0x5403, 4),
@@ -273,6 +275,8 @@ static DEFINE_PCI_DEVICE_TABLE(cxgb4_pci_tbl) = {
  CH_DEVICE(0x5411, 4),
  CH_DEVICE(0x5412, 4),
  CH_DEVICE(0x5413, 4),
+ CH_DEVICE(0x5414, 4),
+ CH_DEVICE(0x5415, 4),
  { 0, }
 };
 
-- 
1.7.1